cdc/capture/capture.go

```diff
@@ -354,6 +369,10 @@ func (c *Capture) AsyncClose() {
 	if c.grpcPool != nil {
 		c.grpcPool.Close()
 	}
+	if c.tableActorSystem != nil {
+		err := c.tableActorSystem.Stop()
+		log.Warn("stop table actor system failed", zap.Error(err))
```

Comment:
Do you think it would be necessary to set `c.tableActorSystem` to `nil`?

Reply:
I think the function `AsyncClose` is assumed to be idempotent.

Reply:
yes, it's better to set  c.tableActorSystem to nil.